dist/index.js

```diff
@@ -5962,7 +6027,7 @@ function runOsX() {
         yield pip.installPython3Dependencies();
         // While rosdep and vcs are available as a Debian package on Ubuntu, they need
         // to be installed through pip on OS X.
-        yield pip.runPython3PipInstall(["rosdep", "vcstool"]);
+        yield pip.runPython3PipInstall(["catkin-pkg", "rosdep", "vcstool"]);
```

Comment:
Seems like this install was missing, but how was this not failing earlier?

Reply:
No, we need to commit the modified file too.